scripts/generate_known_standard_library.py

```diff
@@ -61,10 +37,9 @@ class FakeApp:
             "sre",
         }
 
-        for module in invdata["py:module"]:
-            root, *_ = module.split(".")
-            if root not in ["__future__", "__main__"]:
-                modules.add(root)
+        for module in stdlib_module_names(f"{major_version}.{minor_version}"):
+            if module not in ["__future__", "__main__"]:
```

Comment:
I don't think `__main__` would ever be reported as stdlib.

Reply:
https://github.com/astral-sh/ruff/pull/11409